vm_callinfo.h

```diff
@@ -199,6 +200,8 @@ vm_ci_dump(const struct rb_callinfo *ci)
 static inline const struct rb_callinfo *
 vm_ci_new_(ID mid, unsigned int flag, unsigned int argc, const struct rb_callinfo_kwarg *kwarg, const char *file, int line)
 {
+    if (kwarg)
+        ((struct rb_callinfo_kwarg *)kwarg)->references++;
```

Comment:
Dangling `if`?

Reply:
Thanks! I see no reason to leave off the `{`/`}` here. 

I am still getting the hang of Ruby C style.